### ISSUE TITLE
Expected array default value for '--formatters'; got "text" (string)

### ISSUE THREAD

**Issue by eprislac** [mmozuras/pronto#208]
Pronto posts 0 comments on PRs that I know have broken style-guide rules set up in rubocop. (run with pronto-rubocop). I have rules for 10-line max methods, 100 char wide lines, have seen PRs pass through lately with no comments posted. Looked at the console output on my CI, and saw this line: 
```
Expected array default value for '--formatters'; got "text" (string)
```
I get that error no matter if I have one or two values for formatters. (I run from a Bash script in Jenkins)

**Comment by mmozuras** [comment]
@eprislac could you post the relevant part of that bash script here? 😄

**Comment by eprislac** [comment]
The following line is the entire bash script (with tokens and PR ids obfuscated), and used to work flawlessly: 
```
GITHUB_ACCESS_TOKEN=<my_access_token> PULL_REQUEST_ID=<my_pr_id> pronto run -f github_status github_pr -c origin/develop
```
I have tried changing `GITHUB_ACCESS_TOKEN` to `PRONTO_GITHUB_ACCESS_TOKEN`, as shown in the updated README, it throws an error - 
```
home/jenkins/.rvm/gems/ruby-2.3.1/gems/octokit-4.6.2/lib/octokit/response/raise_error.rb:16:in `on_complete': GET https://api.github.com/repos/CINBCUniversal/sms/pulls?per_page=100: 404 - Not Found // See: https://developer.github.com/v3 (Octokit::NotFound)
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/faraday-0.11.0/lib/faraday/response.rb:9:in `block in call'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/faraday-0.11.0/lib/faraday/response.rb:61:in `on_complete'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/faraday-0.11.0/lib/faraday/response.rb:8:in `call'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/octokit-4.6.2/lib/octokit/middleware/follow_redirects.rb:73:in `perform_with_redirection'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/octokit-4.6.2/lib/octokit/middleware/follow_redirects.rb:61:in `call'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/faraday-0.11.0/lib/faraday/rack_builder.rb:139:in `build_response'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/faraday-0.11.0/lib/faraday/connection.rb:377:in `run_request'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/faraday-0.11.0/lib/faraday/connection.rb:140:in `get'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/sawyer-0.8.1/lib/sawyer/agent.rb:94:in `call'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/octokit-4.6.2/lib/octokit/connection.rb:154:in `request'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/octokit-4.6.2/lib/octokit/connection.rb:84:in `paginate'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/octokit-4.6.2/lib/octokit/client/pull_requests.rb:20:in `pull_requests'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/pronto-0.7.1/lib/pronto/github.rb:89:in `pull_requests'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/pronto-0.7.1/lib/pronto/github.rb:82:in `pull'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/pronto-0.7.1/lib/pronto/github.rb:69:in `pull_id'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/pronto-0.7.1/lib/pronto/github.rb:11:in `pull_comments'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/pronto-0.7.1/lib/pronto/formatter/pull_request_formatter.rb:5:in `existing_comments'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/pronto-0.7.1/lib/pronto/formatter/git_formatter.rb:7:in `format'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/pronto-0.7.1/lib/pronto.rb:59:in `block in run'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/pronto-0.7.1/lib/pronto.rb:58:in `each'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/pronto-0.7.1/lib/pronto.rb:58:in `run'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/pronto-0.7.1/lib/pronto/cli.rb:54:in `block in run'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/pronto-0.7.1/lib/pronto/cli.rb:53:in `chdir'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/pronto-0.7.1/lib/pronto/cli.rb:53:in `run'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/thor-0.19.4/lib/thor/command.rb:27:in `run'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/thor-0.19.4/lib/thor/invocation.rb:126:in `invoke_command'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/thor-0.19.4/lib/thor.rb:369:in `dispatch'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/thor-0.19.4/lib/thor/base.rb:444:in `start'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/gems/pronto-0.7.1/bin/pronto:6:in `<top (required)>'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/bin/pronto:23:in `load'
11:41:28 	from /home/jenkins/.rvm/gems/ruby-2.3.1/bin/pronto:23:in `<main>'
```

I have the following runners installed and none of them are currently posting messages: 
``` 
 - Rubocop
 - Reek
 - Flay
 - Rails Best Practices
 - Fasterer
 - Brakeman
```

**Comment by mmozuras** [comment]
@eprislac I see that you're using `0.7.1`. You should update to a later version. I've checked [our CHANGELOG](https://github.com/mmozuras/pronto/blob/master/CHANGELOG.md) and now see that [this bug](https://github.com/mmozuras/pronto/pull/195) was fixed in `0.8.0` (latest version is `0.8.2`).

**Comment by eprislac** [comment]
Updated to latest. no longer getting errors with new syntax. Everything working correctly now, thanks for the assist!.
